systest/Makefile

```diff
@@ -93,8 +93,8 @@ TESTENVLOGDIR := $(RESULTS_DIR)/f5-openstack-agent_$(SANITIZED_BRANCH)/testenv
 #    Tests use tox for python variants.
 setup:
 	@echo executing $@
-	sudo -E -H $(SCRIPT_DIR)/install_test_infra.sh ;\
-	mkdir -p $(TESTENVLOGDIR)
+	mkdir -p $(TESTENVLOGDIR) && \
+	sudo -E pip install -e $(PROJDIR)
```

Comment:
This guarantees that we're actually testing against the code we Jenkins checked out for us!